From 4236586987b4c1f4241da62dc56c25f8b8c9030c Mon Sep 17 00:00:00 2001
From: John Ralls <jralls@ceridwen.us>
Date: Mon, 22 Jun 2020 13:29:11 -0700
Subject: [PATCH] Never recompile.

---
 libguile/load.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libguile/load.c b/libguile/load.c
index c209812dc..725fe5ae3 100644
--- a/libguile/load.c
+++ b/libguile/load.c
@@ -570,7 +570,7 @@ compiled_is_fresh (SCM full_filename, SCM compiled_filename,
       scm_puts ("\n", scm_current_warning_port ());
     }
 
-  return compiled_is_newer;
+  return 1;
 }
 
 static SCM
-- 
2.26.2

