diff -ru gnucash-docs-3.6-org/guide/C/Makefile.in gnucash-docs-3.6/guide/C/Makefile.in
--- gnucash-docs-3.6-org/guide/C/Makefile.in	2019-06-30 02:22:12.000000000 +0200
+++ gnucash-docs-3.6/guide/C/Makefile.in	2019-08-17 18:02:32.805512092 +0200
@@ -342,7 +342,7 @@
 docname = gnucash-guide
 lang = C
 omffile = $(docname)-C.omf
-entities = \
+entities = $(top_srcdir)/docbook/gnc-docbookx.dtd \
 	fdl-appendix.xml \
 	legal.xml \
 	ch_oview.xml \
Only in gnucash-docs-3.6/guide/C: Makefile.in.bak
diff -ru gnucash-docs-3.6-org/guide/de/Makefile.in gnucash-docs-3.6/guide/de/Makefile.in
--- gnucash-docs-3.6-org/guide/de/Makefile.in	2019-06-30 02:22:12.000000000 +0200
+++ gnucash-docs-3.6/guide/de/Makefile.in	2019-08-17 18:02:32.811512220 +0200
@@ -340,7 +340,7 @@
 docname = gnucash-guide
 lang = de
 omffile = $(docname)-de.omf
-entities = \
+entities = $(top_srcdir)/docbook/gnc-docbookx.dtd \
  ch_accts.xml \
  ch_basics.xml \
  ch_budgets.xml \
Only in gnucash-docs-3.6/guide/de: Makefile.in.bak
Only in gnucash-docs-3.6/guide/it: Makefile.in.bak
diff -ru gnucash-docs-3.6-org/guide/ja/Makefile.in gnucash-docs-3.6/guide/ja/Makefile.in
--- gnucash-docs-3.6-org/guide/ja/Makefile.in	2019-06-30 02:22:12.000000000 +0200
+++ gnucash-docs-3.6/guide/ja/Makefile.in	2019-08-17 18:02:32.822512454 +0200
@@ -341,7 +341,7 @@
 lang = ja
 fopconfig = fop.xconf
 omffile = $(docname)-ja.omf
-entities = \
+entities = $(top_srcdir)/docbook/gnc-docbookx.dtd \
 	fdl-appendix.xml \
 	legal.xml \
 	ch_oview.xml \
Only in gnucash-docs-3.6/guide/ja: Makefile.in.bak
diff -ru gnucash-docs-3.6-org/guide/pt/Makefile.in gnucash-docs-3.6/guide/pt/Makefile.in
--- gnucash-docs-3.6-org/guide/pt/Makefile.in	2019-06-30 02:22:12.000000000 +0200
+++ gnucash-docs-3.6/guide/pt/Makefile.in	2019-08-17 18:02:32.827512560 +0200
@@ -342,7 +342,7 @@
 docname = gnucash-guide
 lang = pt
 omffile = $(docname)-pt.omf
-entities = \
+entities = $(top_srcdir)/docbook/gnc-docbookx.dtd \
 	fdl-appendix.xml \
 	legal.xml \
 	ch_oview.xml \
Only in gnucash-docs-3.6/guide/pt: Makefile.in.bak
diff -ru gnucash-docs-3.6-org/guide/ru/Makefile.in gnucash-docs-3.6/guide/ru/Makefile.in
--- gnucash-docs-3.6-org/guide/ru/Makefile.in	2019-06-30 02:22:12.000000000 +0200
+++ gnucash-docs-3.6/guide/ru/Makefile.in	2019-08-17 18:02:32.831512645 +0200
@@ -341,7 +341,7 @@
 lang = ru
 fopconfig = fop.xconf
 omffile = $(docname)-ru.omf
-entities = \
+entities = $(top_srcdir)/docbook/gnc-docbookx.dtd \
     ch_oview.xml \
     ch_basics.xml \
     ch_accts.xml \
Only in gnucash-docs-3.6/guide/ru: Makefile.in.bak
diff -ru gnucash-docs-3.6-org/help/C/Makefile.in gnucash-docs-3.6/help/C/Makefile.in
--- gnucash-docs-3.6-org/help/C/Makefile.in	2019-06-30 02:22:12.000000000 +0200
+++ gnucash-docs-3.6/help/C/Makefile.in	2019-08-17 18:02:32.836512752 +0200
@@ -342,7 +342,7 @@
 docname = gnucash-help
 lang = C
 omffile = $(docname)-C.omf
-entities = \
+entities = $(top_srcdir)/docbook/gnc-docbookx.dtd \
 	Help_ch_Intro.xml \
         Help_ch_GettingHelp.xml \
 	Help_ch_GettingStarted.xml \
Only in gnucash-docs-3.6/help/C: Makefile.in.bak
diff -ru gnucash-docs-3.6-org/help/de/Makefile.in gnucash-docs-3.6/help/de/Makefile.in
--- gnucash-docs-3.6-org/help/de/Makefile.in	2019-06-30 02:22:13.000000000 +0200
+++ gnucash-docs-3.6/help/de/Makefile.in	2019-08-17 18:02:32.840512837 +0200
@@ -340,7 +340,7 @@
 docname = gnucash-help
 lang = de
 omffile = $(docname)-de.omf
-entities = \
+entities = $(top_srcdir)/docbook/gnc-docbookx.dtd \
 	Help_ch_Intro.xml \
 	Help_ch_GettingStarted.xml \
 	Help_fdl-appendix.xml \
Only in gnucash-docs-3.6/help/de: Makefile.in.bak
Only in gnucash-docs-3.6/help/it: Makefile.in.bak
diff -ru gnucash-docs-3.6-org/help/pt/Makefile.in gnucash-docs-3.6/help/pt/Makefile.in
--- gnucash-docs-3.6-org/help/pt/Makefile.in	2019-06-30 02:22:13.000000000 +0200
+++ gnucash-docs-3.6/help/pt/Makefile.in	2019-08-17 18:02:32.849513028 +0200
@@ -340,7 +340,7 @@
 docname = gnucash-help
 lang = pt
 omffile = $(docname)-pt.omf
-entities = \
+entities = $(top_srcdir)/docbook/gnc-docbookx.dtd \
 	Help_ch_Intro.xml \
         Help_ch_GettingHelp.xml \
 	Help_ch_GettingStarted.xml \
Only in gnucash-docs-3.6/help/pt: Makefile.in.bak
